src/test/scala/com/microsoft/hyperspace/index/E2EHyperspaceRulesTests.scala

```diff
@@ -137,13 +162,35 @@ class E2EHyperspaceRulesTests extends HyperspaceSuite with SQLHelper {
     verifyIndexUsage(query, Seq(getIndexFilesPath(indexConfig.indexName)))
   }
 
-  test("E2E test for join query.") {
-    val leftDf = spark.read.parquet(sampleParquetDataLocation)
+  test(
+    "E2E test for filter query when all columns are selected on partitioned data with lineage.") {
+    spark.conf.set(IndexConstants.INDEX_LINEAGE_ENABLED, "true")
+
+    val df = spark.read.parquet(samplePartitionedParquetDataLocation)
+    val indexConfig =
+      IndexConfig("filterIndex", Seq("imprs", "Query"), Seq("Date", "RGUID", "clicks"))
```

Comment:
Is it possible to use `c1`, `c2`, `c3`, etc. similar to other tests in this suite?

Reply:
Fixed it.